app/models/attendance.rb

```diff
@@ -23,7 +23,21 @@ class Attendance < UuidApplicationRecord
   private
 
   def calc_total_time_in_care
-    self.total_time_in_care = (check_out.nil? || check_in.nil? ? 0 : check_out - check_in)
+    self.total_time_in_care = if check_in && check_out
+                                check_out - check_in
+                              elsif child_approval.child.state == 'NE'
+                                calculate_from_schedule
+                              else
+                                0.seconds
+                              end
+  end
+
+  def calculate_from_schedule
+    child_schedule ? child_schedule.end_time.on(check_in.to_date) - check_in : 8.hours
```

Comment:
@katelovescode Is this using the chlid's end time in the schedule to fill-in a missing end time?  If so, this could work, but currently we've assumed the total scheduled number of hours rather than using real check-in and scheduled check-out.